components/multipart-stream-builder.rst

```diff
@@ -0,0 +1,62 @@
+Multipart Stream Builder
+========================
+
+A multipart stream is a special kind of stream that could be used when transferring files over HTTP. There is currently no PSR-7 support for multipart streams as they are considered to be normal streams with a special content. A multipart stream HTTP request may look like this:
```

Comment:
s/that could be used when transferring files over HTTP/that is used to transfer files over HTTP/